src/test/java/de/tum/in/www1/artemis/service/connectors/LtiServiceTest.java

```diff
@@ -65,11 +73,22 @@
 
     private LtiOutcomeUrl ltiOutcomeUrl;
 
+    private HttpClient httpClient;
+
+    private HttpResponse httpResponse;
+
+    private StatusLine statusLine;
+
     @BeforeEach
     public void init() {
         MockitoAnnotations.openMocks(this);
         SecurityContextHolder.clearContext();
+        httpClient = mock(HttpClient.class);
```

Comment:
Please find a different solution for mocking. This is not consistent with other tests.

Reply:
there is no other scenario in the code with mocking a http client or what are u referring to?

Reply:
removed